src/components/AddressSearch/index.tsx

```diff
@@ -392,10 +288,8 @@ function AddressSearch({
 
     const listEmptyComponent = useCallback(
         () =>
-            network.isOffline || !isTyping ? null : (
-                <Text style={[styles.textLabel, styles.colorMuted, styles.pv4, styles.ph3, styles.overflowAuto]}>{translate('common.noResultsFound')}</Text>
-            ),
-        [network.isOffline, isTyping, styles, translate],
+            !!isOffline && !!isTyping ? <Text style={[styles.textLabel, styles.colorMuted, styles.pv4, styles.ph3, styles.overflowAuto]}>{translate('common.noResultsFound')}</Text> : null,
```

Comment:
Why the logic was updated? This condition doesn't work the same way as before. For example if both `isOffline` and `isTyping` are `true`. Before it returned `null` in this case, now `Text`

Reply:
Corrected